sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -81,36 +78,39 @@
           },
           "order": {
             "title": "Position",
-            "description": "Position of the file",
+            "description": "Position of the file, The lowest position means file is the main file.",
             "type": "integer",
             "default": 1,
             "minimum": 1
           },
           "external_url": {
             "title": "URL to file",
-            "description": "URL to file hosted in an external platform.",
             "type": "string",
             "minLength": 1,
-            "pattern": "^https?://"
+            "pattern": "^https?://.*"
           },
           "restricted": {
             "title": "Restricted",
+            "description": "Type of restriction. 'organisation': the file is only available inside organisation view. 'internal': file is only available with a referenced IP.",
             "type": "string",
-            "minLength": 1
+            "minLength": 1,
+            "pattern": "^(organisation|internal)$"
           },
           "embargo_date": {
             "title": "Embargo date",
+            "description": "Example: 2019-05-05",
             "type": "string",
             "minLength": 1,
-            "format": "date"
```

Comment:
why not keeping this? It can be used with a `pattern` property.

Reply:
Yes you're right!